cmd/test_email_templates/main.go

```diff
@@ -164,6 +164,7 @@ func sendITGovEmails(ctx context.Context, client *email.Client) {
 	intakeID := uuid.New()
 	lifecycleID := "123456"
 	lifecycleExpiresAt := time.Now().AddDate(30, 0, 0)
+	lifecycleIssuedAt := time.Now()
```

Comment:
May as well change `SendIssueLCIDNotification` to match.

Reply:
Resolved in ce45f49ed

Reply:
ce45f49ed resolves this!